olio/models/version.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type Version struct {
+	SkeletonVersion string `json:"skeletonVersion"`
```

Comment:
i thought i had changed this to something like frameworkVersion already?

Reply:
see original

-type Version struct {		
 -	SkeletonVersion string `json:"serviceFrameworkVersion"`		
 -	AppVersion      string `json:"appVersion"`		
 -}		
 -